lib/LANraragi/Plugin/Metadata/MEMS.pm

```diff
@@ -0,0 +1,99 @@
+package LANraragi::Plugin::Metadata::MEMS;

```

Comment:
Add this namespace to the test in `modules.t` so it can be checked during CI.  
I don't think this plugin needs other specific testing otherwise.